texastribune/donations
# Remove libsass

#### What's this PR do?
+ Removes `libsass` and replaces it with a Webpack Sass build (using `node-sass` behind the scenes)
+ Deletes some unused templates

#### Why are we doing this? How does it help us?
`libsass` was slowing down our builds significantly.

#### How should this be manually tested?
+ Build a fresh container
+ `make restart`
+ Ensure `/circle`, `/donate`, `/blastform`, `/business`, `/error` and a non-existent URL all load properly with styles.
+ Do test payments on `/donate` and `/blastform` and ensure both charge pages load properly with styles and that the transactions go through.
+ Force the error page to appear (you could remove the `not` [here](https://github.com/texastribune/donations/blob/master/app.py#L321)) and ensure it loads properly with styles.

#### How should this change be communicated to end users?
NA.

#### Are there any smells or added technical debt to note?
Using Webpack for CSS can be a little confusing at first, but I've documented it in the wiki. And it beats having two different Sass dependencies in the repo, especially when the Python one is really slow.

#### What are the relevant tickets?
https://3.basecamp.com/3098728/buckets/736178/todos/1437016052

#### Have you done the following, if applicable:
* [ ] Added automated tests? *(NA)*
